.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+## Shiny new `go-centrifuge` PR !!!
+
+This is intended as a guide when describing/reviewing pull requests for `go-centrifuge` or related projects
+
+- [ ] Issue link
+- [ ] The solution implemented
+- [ ] If this is only part of the solution what is that part?
```

Comment:
Not sure if we really need point 2 and 3 IMO

Reply:
its needed as its sometimes not obvious what the fix is and also which part of the fix it is. As described this is just a guide not a requirement.